bundles/com.espressif.idf.swt.custom/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,19 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: %Bundle-Name
+Bundle-SymbolicName: com.espressif.idf.swt.custom;singleton:=true
+Bundle-Version: 1.0.1.qualifier
+Export-Package: com.espressif.idf.swt.custom
+Bundle-Activator: com.espressif.idf.swt.custom.Activator
+Bundle-Vendor: %Bundle-Vendor
+Bundle-RequiredExecutionEnvironment: JavaSE-17
+Automatic-Module-Name: com.espressif.idf.swt.custom
+Import-Package: org.osgi.framework;version="1.3.0"
+Require-Bundle: org.eclipse.swt,
+ org.eclipse.jface,
+ org.eclipse.core.variables,
+ org.eclipse.equinox.registry,
+ com.espressif.idf.core,
```

Comment:
Do we need **com.espressif.idf.core** for this? If possible, avoid this.

Reply:
Hi @kolipakakondal, we are using Logger from the core plugin to track the exceptions. I think it's ok, since we don't need any custom swt elements in the core plugin